.github/workflows/ci.yml

```diff
@@ -470,14 +448,11 @@ jobs:
           name: "saw-Linux-${{ matrix.ghc }}"
           path: ./s2nTests/bin
 
-      - uses: actions/download-artifact@v2
-        with:
-          path: ./s2nTests/bin
-          name: abc-${{ runner.os }}
-
       - shell: bash
         working-directory: s2nTests
         run: |
+          curl -o solvers-bin.zip -sL "https://github.com/GaloisInc/what4-solvers/releases/download/$SOLVER_PKG_VERSION/ubuntu-18.04-bin.zip"
```

Comment:
It unfortunately installs them in a different place. I'd like to refactor things so that's not true anymore, but for now I'm trying to make somewhat minimal changes.